dev/sample_mqttplugin/__init__.py

```diff
@@ -104,6 +119,13 @@ def parse_item(self, item):
                         with the item, caller, source and dest as arguments and in case of the knx plugin the value
                         can be sent to the knx with a knx write function within the knx plugin.
         """
+        # check for alive item
+        if item.property.path == self._pause_item_path:
+            self.logger.debug(f'alive item {item.property.path} registered')
```

Comment:
Should this rather be calles pause item here, too..?

Reply:
yes.